index.rst

```diff
@@ -48,10 +48,63 @@
 
    **This technote is not yet published.**
 
-   This proposal outlines how observing tasks will be flowed from the planning stage through execution, then linked to data analysis technotes and/or notebooks.
+   This proposal outlines how observing tasks will be flowed from the planning stage through execution,
+   then linked to data analysis technotes and/or notebooks.
+
+Motivation for this proposal
+============================
+
+    * We want central repository for all information related to a task executed at night.
+    * We want to have a way to know how and when the tasks were run and how the data was processed.
+    * Search and find tasks and their results.
+
+Steps
+=====
+
+    * Run planning
+        * Which tasks/test are we running during a observation run?
```

Comment:
Night plans need to point to discrete tasks that are linked to where the procedure and artifacts are archived.
The plan then dictates required conditions and constraints on observations

Reply:
Will those be defined just on photometric or no? Or are we looking to start deploying an IQ system?

Reply:
It'll go beyond photometric/IQ I imagine. Airmass will matter, even moon angle at some level... but I don't think we need to flush out those details here.